src/wallet/CarbonWallet.ts

```diff
@@ -856,9 +856,7 @@ export class CarbonWallet {
     const txGasPrices = this.gasFee?.txGasPrices ?? {}
 
     if (!txGasPrices[this.defaultFeeDenom]) {
-      const queryClient = this.getQueryClient()
-      const { minGasPrices } = await queryClient.fee.MinGasPriceAll({});
-      const newDefaultFeeDenom = minGasPrices[0]?.denom;
+      const newDefaultFeeDenom = Object.keys(txGasPrices)[0]
```

Comment:
could you remove the MinGasPriceAll query for `KeplrAccount.ts` and `LeapAccount.ts` as well? since its already stored on an initialized sdk shouldn't need to query again
![image](https://github.com/Switcheo/carbon-js-sdk/assets/77280201/14fe6932-8ddb-4ab8-b004-6621fcb27130)

![image](https://github.com/Switcheo/carbon-js-sdk/assets/77280201/54199107-1a21-4b7e-81d3-2f249a0ed4cb)

within the Keplr and Leap Providers, `getChainId` which is already queried when sdk is initialized is also called again pls help to remove, prolly can just change SDKProvider to CarbonSDK since sdk is passed in as the param anyway
![image](https://github.com/Switcheo/carbon-js-sdk/assets/77280201/7873e367-12e8-4768-aa19-5a1d4f126b1d)

do yarn link with disco to test, might have to fix some typing on demex from the gov commit